spec/lib/hubspot/company_properties_spec.rb

```diff
@@ -14,158 +14,325 @@
     end
   end
 
-  let(:example_groups) do
-    VCR.use_cassette('company_properties/groups_example') do
-      HTTParty.get('https://api.hubapi.com/companies/v2/groups?hapikey=demo').parsed_response
-    end
-  end
+  before { Hubspot.configure(hapikey: ENV.fetch("HUBSPOT_HAPI_KEY", "demo")) }
```

Comment:
This is something I've been meaning to do for a while in addition to having a test application setup for tests run in Travis CI. This way the tests won't break if someone is doing something funky in the demo application. We should set this up for all tests possibly in the RSpec configuration block.

Reply:
I think we can avoid setting up a test application for Travis CI. The tests, when ran by Travis or ran locally with VCR_RECORD turned off, we can use a fake key value, like `fake-123`.

I agree it would be great to avoid using the "demo" account when recording API interactions as the demo account returns more data than we want to store in a VCR file. 

I'd like to update the tests to use the Oauth flow as the HubSpot API docs recommend using an access token from the OAuth flow as the preferred way to authenticate. I don't know if I'll have time to reach this goal but just mentioning it here as food for thought.